2-crud-api/src/index.ts

```diff
@@ -2,17 +2,19 @@ import dotenv from 'dotenv';
 dotenv.config();
 import express, { Express } from 'express';
 
-import { pgInit } from './loaders/pgInit';
+// import { pgInit } from './loaders/pgInit';
 import { userRouter } from './controllers/userRouter';
+import { groupRouter } from './controllers/groupRouter';
 
 
 async function start() {
-  await pgInit();
+  // await pgInit();
 
   const app: Express = express();
   const port = process.env.PORT;
   app.use(express.json());
   app.use(userRouter);
+  app.use(groupRouter);
```

Comment:
```suggestion
  app.use('/user/', userRouter);
  app.use('/group/', groupRouter);
```
This fits the REST architecture better